config/settings.js

```diff
@@ -1,23 +1,23 @@
 module.exports = {
-    basePath: 'http://localhost:3000/',
-    googleAuth: {
-        clientID: '701417346516-33qb38qk06480vh9vg91btfletf6nlqr.apps.googleusercontent.com',
-        clientSecret: '45rNBMN0HsoYJmsyxZ3qkHeC',
-        callbackURL: 'http://localhost:3000/auth/google/callback',
-        access_type: 'offline'
-    },
-    secretKey: 'pcprepkittokengenkey',
-    nodeMailer: {
-        PROVIDER: 'Gmail',
-        EMAIL: 'pcprepkit@gmail.com',
-        PASSWORD: 'pcprepkit9'
-    },
-    apiai: {
-        clientToken: '2da698372070440e9ac9ac5c96917147'
-    },
-    databaseSetup: {
-        DATABASE_NAME: 'pcprepkit',
-        USERNAME: 'pcprepkit',
-        PASSWORD: 'pcprepkit'
-    }
+  basePath: 'http://localhost:3000/',
+  googleAuth: {
+    clientID: '311448429695-va7g4f7takj7blpudu4gtvf9qeqr86a0.apps.googleusercontent.com',
+    clientSecret: 'YG0pAqQwrR8Lsdyh89cFHg3x',
+    callbackURL: 'http://localhost:3000/auth/google/callback',
+    access_type: 'offline'
+  },
+  secretKey: 'pcprepkittokengenkey',
+  nodeMailer: {
+    PROVIDER: 'Gmail',
+    EMAIL: 'pcprepkit@gmail.com',
+    PASSWORD: 'pcprepkit9'
+  },
+  apiai: {
+    clientToken: '0f67eb54f1d74ea4a3d9c34a76da68ca'
+  },
```

Comment:
Again, was there a need to change the `clientToken`? Please revert this change if it was not necessary